docs/en/Configuring-Alluxio-with-S3.md

```diff
@@ -220,5 +220,5 @@ If you want to share the S3 mount point with other users in Alluxio namespace, y
 
 ### Permission change
 
-In addition, chown/chgrp/chmod to Alluxio directories and files do NOT propagate to the underlying
+'chown', 'chgrp', and 'chmod' of Alluxio directories and files do NOT propagate to the underlying
```

Comment:
can you change to 
<pre>`chown`, `chgrp`, and `chmod`</pre>
rather than 
<pre>'chown', 'chgrp', and 'chmod'</pre>

Reply:
I have changed it.